internal/services/notification/notification_test.go

```diff
@@ -539,3 +544,117 @@ func createCommitStatusDelivery(t *testing.T, ctx context.Context, ns *Notificat
 
 	return delivery
 }
+
+func TestGetProjectRunWebhookDeliveries(t *testing.T) {
+	t.Parallel()
+
+	dir := t.TempDir()
+	log := testutil.NewLogger(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	ns := setupNotificationService(ctx, t, log, dir)
+
+	t.Logf("starting ns")
+
+	time.Sleep(1 * time.Second)
+
+	runWebhooks := make([]*types.RunWebhook, 10)
+	runWebhookDeliveriesDelivered := make([]*types.RunWebhookDelivery, 20)
+	runWebhookDeliveriesNotDelivered := make([]*types.RunWebhookDelivery, 20)
+	for i := 0; i < len(runWebhooks); i++ {
+		runWebhooks[i] = createRunWebhook(t, ctx, ns, project01)
+		runWebhookDeliveriesDelivered[i] = createRunWebhookDelivery(t, ctx, ns, runWebhooks[i].ID, types.DeliveryStatusDelivered)
+		runWebhookDeliveriesNotDelivered[i] = createRunWebhookDelivery(t, ctx, ns, runWebhooks[i].ID, types.DeliveryStatusNotDelivered)
+	}
+
+	for i := 0; i < len(runWebhooks); i++ {
+		runWebhooks[i] = createRunWebhook(t, ctx, ns, project02)
+		createRunWebhookDelivery(t, ctx, ns, runWebhooks[i].ID, types.DeliveryStatusDelivered)
+		createRunWebhookDelivery(t, ctx, ns, runWebhooks[i].ID, types.DeliveryStatusNotDelivered)
+	}
+
+	t.Run("test get run webhook deliveries with limit = 0", func(t *testing.T) {
```

Comment:
Looks like you forgot to add `test get ... with limit less than ... continuation`